### PR TITLE
cloud: update for the cluster creation UI

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)
cloud: update for the cluster creation UI

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-build-cluster-in-cloud.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ If you need to run TiDB on your local machine, see [Starting TiDB Locally](/quic
 
 1. If you do not have a TiDB Cloud account, click [TiDB Cloud](https://tidbcloud.com/free-trial) to sign up for an account.
 2. [Sign in](https://tidbcloud.com/) with your TiDB Cloud account.
-3. To create a Developer Tier cluster for one year free, you can either select the **Developer Tier** plan on the [plan page](https://tidbcloud.com/console/plans) or click [Create a Cluster (Dev Tier)](https://tidbcloud.com/console/create-cluster?tier=dev).
-4. On the **Create a Cluster (Dev Tier)** page, set up your cluster name, password, cloud provider (for now, only AWS is available for Developer Tier), and region (a nearby region is recommended). Then click **Create** to create your cluster.
+3. To create a Developer Tier cluster for one year free, you can either select the **Developer Tier** plan on the [plan page](https://tidbcloud.com/console/plans) or click [Create Now](https://tidbcloud.com/console/create-cluster?tier=dev) and select **Developer Tier** on the **Create a Cluster** page.
```
Comment (TomShawn) [inline develop/dev-guide-build-cluster-in-cloud.md:18]:
```suggestion
3. To create a Developer Tier cluster for one year free, you can either select the **Developer Tier** plan on the [plan page](https://tidbcloud.com/console/plans) or click [Create Cluster](https://tidbcloud.com/console/create-cluster?tier=dev) and select **Developer Tier** on the **Create a Cluster** page.
```

**Review by zhoubasten — APPROVED** [review]
LGTM

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 2]
```diff
@@ -16,8 +16,13 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
...
 1. In the TiDB Cloud Console, get the TiDB Cloud account ID and external ID of the target TiDB cluster.
 
-    1. In the TiDB Cloud Console, choose your target project, and then click the name of your target cluster to go to its overview page.
-    2. In the upper-right corner, click **Import Data**. The **Data Import Task** page is displayed.
+    1. In the TiDB Cloud Console, choose your target project, and navigate to the **Active Clusters** page.
+    2. Find the area of your target cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
+
+        > **Tip:**
+        >
+        > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/config-s3-and-gcs-access.md:24]:
```suggestion
        > Alternatively, you can also click the name of your cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 3]
```diff
@@ -18,9 +18,13 @@ After your TiDB cluster is created on TiDB Cloud, you can use one of the followi
 
 To connect to your TiDB cluster via standard connection, perform the following steps:
 
-1. Navigate to the **Active Clusters** page and click the name of your newly created cluster.
+1. Navigate to the **Active Clusters** page.
 
-2. Click **Connect** in the upper-right corner. A connection dialog box is displayed.
+2. Find the area of your newly created cluster, and click **Connect** in the upper-right corner of the area. A connection dialog box is displayed.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:23]:
```suggestion
2. Find the area of your cluster, and click **Connect** in the upper-right corner of the area. A connection dialog box is displayed.
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 4]
```diff
@@ -18,9 +18,13 @@ After your TiDB cluster is created on TiDB Cloud, you can use one of the followi
...
+
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:27]:
```suggestion
    > Alternatively, you can also click the name of your cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 5]
```diff
@@ -49,9 +53,13 @@ To connect to your TiDB cluster via standard connection, perform the following s
 
 To connect to your TiDB cluster via VPC peering, perform the following steps:
 
-1. Navigate to the **Active Clusters** page and click the name of your newly created cluster.
+1. Navigate to the **Active Clusters** page.
+
+2. Find the area of your newly created cluster, click **Connect** in the upper-right corner of the area, and select the **VPC Peering** tab in the connection dialog.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:58]:
```suggestion
2. Find the area of your cluster, click **Connect** in the upper-right corner of the area, and select the **VPC Peering** tab in the connection dialog.
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 6]
```diff
@@ -49,9 +53,13 @@ To connect to your TiDB cluster via standard connection, perform the following s
...
 
-2. Click **Connect** in the upper-right corner, and select the **VPC Peering** tab at the connection dialog.
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:62]: ditto

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 7]
```diff
@@ -69,9 +77,13 @@ To connect to your TiDB cluster via VPC peering, perform the following steps:
 
 To connect to your TiDB cluster using SQL shell, perform the following steps:
 
-1. Navigate to the **Active Clusters** page and click the name of your newly created cluster.
+1. Navigate to the **Active Clusters** page.
+
+2. Find the area of your newly created cluster, click **Connect** in the upper-right corner of the area, and select the **Web SQL Shell** tab in the connection dialog.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:82]: ditto

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 8]
```diff
@@ -69,9 +77,13 @@ To connect to your TiDB cluster via VPC peering, perform the following steps:
...
 
-2. Click **Connect** in the upper-right corner, and select the **Web SQL Shell** tab at the connection dialog.
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:86]: ditto

File: tidb-cloud/import-sample-data.md [hunk 9]
```diff
@@ -7,9 +7,12 @@ summary: Learn how to import sample data into TiDB Cloud via UI.
 
 This document describes how to import sample data into TiDB Cloud via the UI. The sample data used is the system data from Capital Bikeshare, released under the Capital Bikeshare Data License Agreement. Before importing the sample data, you need to have one TiDB cluster.
 
-1. Navigate to the **Active Clusters** page and click the name of your newly created cluster. The overview page of your cluster is displayed.
+1. Navigate to the **Active Clusters** page.
+2. Find the area of your newly created cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:11]:
```suggestion
2. Find the area of your cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
```

File: tidb-cloud/import-sample-data.md [hunk 10]
```diff
@@ -7,9 +7,12 @@ summary: Learn how to import sample data into TiDB Cloud via UI.
...
 
-2. In the upper-right corner, click **Import Data**. The **Data Import Task** page is displayed.
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/import-sample-data.md:15]:
```suggestion
    > Alternatively, you can also click the name of your cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```

File: tidb-cloud/configure-security-settings.md [hunk 11]
```diff
@@ -10,7 +10,7 @@ When you create a cluster, the TiDB Cloud console will automatically guide you t
 If you do not configure the root password or allowed IP addresses at that time, or if you want to modify the cluster security settings, take the following steps:
 
 1. In the TiDB Cloud console, navigate to the **Active Clusters** page of your project, and then locate the pane of your cluster.
```
Comment (qiancai) [inline tidb-cloud/configure-security-settings.md:12]:
```suggestion
1. In the TiDB Cloud console, navigate to the **Active Clusters** page of your project, and then find the area of your cluster.
```

File: tidb-cloud/configure-security-settings.md [hunk 12]
```diff
@@ -10,7 +10,7 @@ When you create a cluster, the TiDB Cloud console will automatically guide you t
...
-2. Click **Security Settings** in the pane. The **Security Settings** dialog is displayed.
+2. Click **Security Settings** in the upper-right corner of the pane. The **Security Settings** dialog is displayed.
```
Comment (qiancai) [inline tidb-cloud/configure-security-settings.md:13]:
```suggestion
2. Click **Security Settings** in the upper-right corner of the area. The **Security Settings** dialog is displayed.
```

File: tidb-cloud/connect-to-tidb-cluster.md (hunk 6 above)
Comment (TomShawn) [reply tidb-cloud/connect-to-tidb-cluster.md:62]:
```suggestion
    > Alternatively, you can also click the name of your cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```

File: tidb-cloud/connect-to-tidb-cluster.md (hunk 7 above)
Comment (TomShawn) [reply tidb-cloud/connect-to-tidb-cluster.md:82]:
```suggestion
2. Find the area of your cluster, click **Connect** in the upper-right corner of the area, and select the **Web SQL Shell** tab in the connection dialog.
```

File: tidb-cloud/connect-to-tidb-cluster.md (hunk 8 above)
Comment (TomShawn) [reply tidb-cloud/connect-to-tidb-cluster.md:86]:
```suggestion
    > Alternatively, you can also click the name of your cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```

File: tidb-cloud/config-s3-and-gcs-access.md [hunk 13]
```diff
@@ -16,8 +16,13 @@ To allow TiDB Cloud to access the source data in your Amazon S3 bucket, take the
...
 1. In the TiDB Cloud Console, get the TiDB Cloud account ID and external ID of the target TiDB cluster.
 
-    1. In the TiDB Cloud Console, choose your target project, and then click the name of your target cluster to go to its overview page.
-    2. In the cluster overview pane on the left, click **Import**. The **Data Import Task** page is displayed.
+    1. In the TiDB Cloud Console, choose your target project, and navigate to the **Active Clusters** page.
+    2. Find the area of your target cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
+
+        > **Tip:**
+        >
+        > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/config-s3-and-gcs-access.md]:
```suggestion
        > Alternatively, you can also click the name of your target cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```

File: tidb-cloud/import-csv-files.md [hunk 14]
```diff
@@ -67,8 +67,13 @@ To allow TiDB Cloud to access the CSV files in the Amazon S3 or GCS bucket, do o
...
 To import the CSV files to TiDB Cloud, take the following steps:
 
-1. Navigate to the TiDB Clusters page and click the name of your target cluster. The overview page of your target cluster is displayed.
-2. In the cluster information pane on the left, click **Import**. The **Data Import Task** page is displayed.
+1. Navigate to the **Active Clusters** page.
+2. Find the area of your target cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
+
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/import-csv-files.md:75]:
```suggestion
    > Alternatively, you can also click the name of your target cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```

File: tidb-cloud/import-parquet-files.md [hunk 15]
```diff
@@ -76,8 +76,13 @@ To allow TiDB Cloud to access the Parquet files in the Amazon S3 or GCS bucket,
...
 To import the Parquet files to TiDB Cloud, take the following steps:
 
-1. Navigate to the TiDB Clusters page and click the name of your target cluster. The overview page of your target cluster is displayed.
-2. In the cluster information pane on the left, click **Import**. The **Data Import Task** page is displayed.
+1. Navigate to the **Active Clusters** page.
+2. Find the area of your target cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
+
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/import-parquet-files.md:84]:
```suggestion
    > Alternatively, you can also click the name of your target cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```

File: tidb-cloud/migrate-from-aurora-bulk-import.md [hunk 16]
```diff
@@ -11,8 +11,13 @@ This document describes how to migrate data from Amazon Aurora MySQL to TiDB Clo
...
 To import data, perform the following steps:
 
-1. Navigate to the TiDB Clusters page and click the name of your target cluster. The overview page of your target cluster is displayed.
-2. In the cluster information pane on the left, click **Import**. The **Data Import Task** page is displayed.
+1. Navigate to the **Active Clusters** page.
+2. Find the area of your target cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
+
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/migrate-from-aurora-bulk-import.md:19]:
```suggestion
    > Alternatively, you can also click the name of your target cluster on the **Active Clusters** page and click **Import Data** in the upper-right corner.
```

File: tidb-cloud/select-cluster-tier.md [hunk 17]
```diff
@@ -51,8 +51,12 @@ Whenever you use or set a database user name, you must include the prefix in the
 
 To get the prefix for your cluster, take the following steps:
 
-1. In the TiDB Cloud console, navigate to the **Active Clusters** page of your project and click the name of your cluster.
-2. In the cluster information pane on the left, click **Connect**. The **Connect to TiDB** dialog is displayed.
+1. Navigate to the **Active Clusters** page.
+2. Find the area of your newly created cluster, and click **Connect** in the upper-right corner of the area. A connection dialog box is displayed.
```
Comment (qiancai) [inline tidb-cloud/select-cluster-tier.md:55]:
```suggestion
2. Click **Connect** in the upper-right corner of the area. A connection dialog box is displayed.
```

File: tidb-cloud/select-cluster-tier.md [hunk 18]
```diff
@@ -51,8 +51,12 @@ Whenever you use or set a database user name, you must include the prefix in the
...
+
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/select-cluster-tier.md:59]:
```suggestion
    > Alternatively, you can also click the name of your cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 19]
```diff
@@ -186,9 +186,12 @@ You can also use the AWS dashboard to configure the VPC peering connection.
 
 ### Step 3: Connect to the TiDB cluster on TiDB Cloud
 
-1. Navigate to the **Active Clusters** page and click the name of your cluster.
+1. Navigate to the **Active Clusters** page.
+2. Find the area of your newly created cluster, and click **Connect** in the upper-right corner of the area. The connection dialog is displayed. You can see the **Status** of the VPC peering is **active**.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:190]:
```suggestion
2. Find the area of your target cluster, and click **Connect** in the upper-right corner of the area. The connection dialog is displayed. You can see the **Status** of the VPC peering is **active**.
```

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 20]
```diff
@@ -186,9 +186,12 @@ You can also use the AWS dashboard to configure the VPC peering connection.
...
 
-2. Click **Connect**. The **Connect to TiDB** dialog displays. You could see the **Status** of the VPC peering is **active**.
+    > **Tip:**
+    >
+    > Alternatively, you can also click the name of your newly created cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```
Comment (qiancai) [inline tidb-cloud/set-up-vpc-peering-connections.md:194]:
```suggestion
    > Alternatively, you can also click the name of your target cluster on the **Active Clusters** page and click **Connect** in the upper-right corner.
```

File: tidb-cloud/tidb-cloud-quickstart.md [hunk 21]
```diff
@@ -117,9 +119,12 @@ To create a Dedicated Tier cluster, take the following steps:
 
 We provide Capital Bikeshare sample data for you to easily import data and run sample queries.
 
-1. Navigate to the **Active Clusters** page and click the name of your newly created cluster. The overview page of your cluster is displayed.
+1. Navigate to the **Active Clusters** page.
+2. Find the area of your newly created cluster and click **Import Data** in the upper-right corner of the area. The **Data Import Task** page is displayed.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-quickstart.md:123]:
```suggestion
2. In the area of your newly created cluster, click **Import Data** in the upper-right corner. The **Data Import Task** page is displayed.
```

**Review by qiancai — APPROVED** [review]
LGTM

**Pull request merged**